exercises/rotational-cipher/rotational_cipher.exs

```diff
@@ -0,0 +1,13 @@
+defmodule RotationalCipher do
+  @doc """
+  Given a plaintext and amount to shift by, return a rotated string.
+
+  Example:
+  iex> RotationalCipher.rotate("Attack at dawn", 13)
+  "Nggnpx ng qnja"
+  """
+  @spec rotate(text :: String.t(), shift :: integer) :: String.t()
+  def rotate(_text, _shift) do
```

Comment:
I'm not sure how I feel about underscoring the variables in the template file. @devonestes ?